src/ProjectService/Startup.cs

```diff
@@ -72,42 +107,34 @@ public void ConfigureServices(IServiceCollection services)
 
         private void ConfigureMassTransit(IServiceCollection services)
         {
-            _rabbitMqConfig = Configuration
-                .GetSection(BaseRabbitMqOptions.RabbitMqSectionName)
-                .Get<RabbitMqConfig>();
-
-            services.AddMassTransit(x =>
+            services.AddMassTransit(busConfigurator =>
             {
-                x.UsingRabbitMq((context, cfg) =>
+                busConfigurator.UsingRabbitMq((context, cfg) =>
                 {
                     cfg.Host(_rabbitMqConfig.Host, "/", host =>
                     {
-                        host.Username($"{_rabbitMqConfig.Username}_{_rabbitMqConfig.Password}");
-                        host.Password(_rabbitMqConfig.Password);
+                        host.Username($"{_serviceInfoConfig.Name}_{_serviceInfoConfig.Id}");
+                        host.Password(_serviceInfoConfig.Id);
                     });
                 });
 
-                RegisterRequestClients(x, _rabbitMqConfig);
+                RegisterRequestClients(busConfigurator);
 
-                x.ConfigureKernelMassTransit(_rabbitMqConfig);
+                busConfigurator.AddRequestClients(_rabbitMqConfig, _logger);
             });
 
             services.AddMassTransitHostedService();
 	    }
 
         private void RegisterRequestClients(
-            IServiceCollectionBusConfigurator busConfigurator,
-            RabbitMqConfig rabbitMqConfig)
+            IServiceCollectionBusConfigurator busConfigurator)
         {
-            busConfigurator.AddRequestClient<IGetFileRequest>(
-                    new Uri($"{rabbitMqConfig.BaseUrl}/{rabbitMqConfig.GetFileEndpoint}"));
-
             busConfigurator.AddRequestClient<IGetUserDataRequest>(
-                new Uri($"{rabbitMqConfig.BaseUrl}/{rabbitMqConfig.GetUserDataEndpoint}"),
+                new Uri($"{_rabbitMqConfig.BaseUrl}/{_rabbitMqConfig.GetUserDataEndpoint}"),
                 RequestTimeout.After(ms: 100));
 
             busConfigurator.AddRequestClient<IGetDepartmentRequest>(
-                new Uri($"{rabbitMqConfig.BaseUrl}/{rabbitMqConfig.GetDepartmentDataEndpoint}"),
+                new Uri($"{_rabbitMqConfig.BaseUrl}/{_rabbitMqConfig.GetDepartmentDataEndpoint}"),
                 RequestTimeout.After(ms: 100));
         }
```

Comment:
Remove this and 5 other private methods

Reply:
I think I've got a brain damage, but done